cylc/flow/network/scan.py

```diff
@@ -35,7 +36,7 @@
 MSG_QUIT = "QUIT"
 MSG_TIMEOUT = "TIMEOUT"
 SLEEP_INTERVAL = 0.01
-
+logging.basicConfig(filename='app.log', format='%(levelname)s:%(message)s',level=logging.DEBUG)
```

Comment:
The build failed on Travis because of this change. The missing space after comma, as well as line length. However, see comment above, I think we can use `cylc.flow.LOG` instead.

Reply:
Done! :)